dev-tools/make/mage.mk

```diff
@@ -7,7 +7,7 @@ export MAGE_IMPORT_PATH
 mage:
 ifndef MAGE_PRESENT
 	@echo Installing mage $(MAGE_VERSION) from vendor dir.
-	@go install -mod=vendor -ldflags="-X $(MAGE_IMPORT_PATH)/mage.gitTag=$(MAGE_VERSION)" ${MAGE_IMPORT_PATH}
+	@go install -mod=vendor -ldflags="-X $(MAGE_IMPORT_PATH)/mage.gitTag=$(MAGE_VERSION)" ${MAGE_IMPORT_PATH}/...
```

Comment:
In the original PR to `master` there is no `@` to suppress the command output. Just curious, why do we do it in `7.x`?

Reply:
I do not know exactly. But I assume it has to do something with Jenkins pipelines. :D @kuisathaverat changed it in https://github.com/elastic/beats/pull/15570, he can tell us more about it.

Reply:
I do not remember it, but I guess it was to see the command, and I push the change 🤦‍♂ so it should have the @

Reply:
Thanks @kuisathaverat. So sounds like the `@` here in this backport PR is what we want. 👍

Reply:
https://github.com/elastic/beats/pull/17466